modules/launcher/src/test/java/org/mule/module/launcher/builder/AbstractArtifactFileBuilder.java

```diff
@@ -255,16 +209,8 @@ private void buildBrokenZipFile(File tempFile) throws IOException
         FileUtils.write(tempFile, "This is content represents invalid compressed data");
     }
 
-    protected static final class ZipResource
+    protected List<ZipResource> getCustomResources() throws Exception
     {
-
-        final String file;
-        final String alias;
-
-        protected ZipResource(String file, String alias)
-        {
-            this.file = file;
-            this.alias = alias;
-        }
+        return new ArrayList<>();
```

Comment:
Is there possibilities in the future that this returns something other than an empty list?
If not, why not just make this method abstract and remove the super call form the extensions?